mkdocs/docs/getting-started/concepts.md

```diff
@@ -4,21 +4,20 @@ template: main.html
 
 # What is Iter8?
 
-Kubernetes release engineering toolkit built for DevOps, MLOps, SRE and data science teams. 
+Kubernetes release optimizer built for DevOps and MLOps teams.
 
 ## What is an Iter8 experiment?
-Iter8 experiments make it simple to collect performance and business metrics for Kubernetes apps and ML models, assess and compare multiple app/ML model versions, safely rollout winning versions, and maximize business value with each release.
+Iter8 experiments make it simple to collect performance and business metrics for apps and ML models, assess, compare and validate multiple app/ML model versions, safely rollout winning version, and maximize business value in each release.
 
-### Example
-The following picture illustrates an Iter8 experiment that performs load testing with SLO validation of a gRPC service.
-
-![Load testing gRPC](../tutorials/load-test-grpc/images/grpc-overview.png)
+<p align='center'>
+  <img alt-text="load-test-http" src="../../images/iter8-intro-dark.png" width="70%" />
+</p>
 
 ### Experiment chart
-In order to enable reuse, Iter8 experiments are templated and packaged as specialized [Helm charts](https://helm.sh/docs/topics/charts/). Experiment charts can be combined with values to generate `experiments.yaml` files that provide fully defined experiment specifications.
+Experiment charts are specialized [Helm charts](https://helm.sh/docs/topics/charts/) that contain reusable experiment templates. Iter8 combines experiment charts with user supplied values to generate runnable `experiment.yaml` files.
 
 #### Iter8 Hub
-Iter8 Hub is a specific location within in the [Iter8 GitHub repo](https://github.com/iter8-tools/iter8) that hosts several pre-packaged and reusable charts. These charts enable to you to launch powerful release engineering experiments in a matter of seconds. Their usage is described in depth in various [Iter8 tutorials](../tutorials/load-test-http/overview.md).
+Iter8 hub is a specific location within in the [Iter8 GitHub repo](https://github.com/iter8-tools/iter8) that hosts several pre-packaged and reusable charts. These charts enable to you to launch powerful release optimization experiments in seconds. Their usage is described in depth in various [Iter8 tutorials](https://iter8.tools/latest/tutorials/load-test-http/overview/).
```

Comment:
The link to the example should be updated.

Reply:
The link that it should be updated with ... will come into existence after we do this merge. Hence, the need for staging it.